lib/services/ngsi/ngsiService.js

```diff
@@ -67,7 +68,8 @@ function init() {
  * @param {String} token            User token to identify against the PEP Proxies (optional).
  */
 function sendUpdateValue(entityName, attributes, typeInformation, token, callback) {
-    entityHandler.sendUpdateValue(entityName, attributes, typeInformation, token, callback);
+    const newCallback = statsRegistry.withStats('updateEntityRequestsOk', 'updateEntityRequestsError', callback);
```

Comment:
only here and raise/release alarm are feeding statsRegistry at this moment. It should be added also for create/delete groups/devices and so on ?

Reply:
I only added metrics for things I want to alert on our monitoring systems. Currently, I want to create alerts:

- If the number of failed updates increases much above average,
- If the number of successful updates falls much below average, 
- if there are active alarms RAISE without RELEASE.

I don't foresee alerting on creation of devices or groups, so I haven't added new stats on that. It might be better to add metrics on demand, when some use case demands them.